extension/content.js

```diff
@@ -26,15 +26,15 @@ function toggleFiles() {
 	let i = 0;
 
 	for (const el of rows) {
-		if (el.querySelector('.content a')) {
-			const fileName = el.querySelector('td.content a').innerText;
+		if (el.querySelector('.content > * > *')) {
```

Comment:
Using star is too loose. Could maybe be `'.content > span > :-webkit-any(a, span)'`

https://css-tricks.com/almanac/selectors/m/matches/